### PR TITLE
Listing: select filter smart fill

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Decisions | - Add smart fill for selects so they always occupy at least 100% of the screen's width. |
| Animated GIF | Below |

https://user-images.githubusercontent.com/24736423/127486619-8676bb84-eeec-4f7d-9e76-49e5bcfce2cf.mov

### CODE REVIEW

**Review by gcandal — APPROVED** [review]

**Comment by joamag** [comment]
@3rdvision let's use the simpler approach of 1, 2 and 3 columns and from there use the 1/3 size of the width per each item

File: react/components/organisms/listing/listing.js [hunk 1]
```diff
@@ -60,10 +60,13 @@ export class Listing extends Component {
             itemsOffset: 0,
             filters: this.props.filtersValue,
             loading: false,
+            searchLoaded: false,
```
Comment (gcandal) [inline react/components/organisms/listing/listing.js:63]: Why is this needed?

Comment (3rdvision) [reply react/components/organisms/listing/listing.js:63]: because it will trigger multiple times if this is not done this way. We already have this pattern in other places in Robin

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]
